pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -171,6 +181,7 @@ func validateKubeSchedulerProfile(path *field.Path, apiVersion string, profile *
 	if len(profile.SchedulerName) == 0 {
 		errs = append(errs, field.Required(path.Child("schedulerName"), ""))
 	}
+	errs = append(errs, validatePercentageOfNodesToScore(profile.PercentageOfNodesToScore)...)
```

Comment:
You should pass a path here.

Reply:
Ok, added `path` to `func validatePercentageOfNodesToScore`